framework/pom.xml

```diff
@@ -450,5 +457,38 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>java</id>
```

Comment:
Can we change the profile name to be more clear? Instead of `java`, may be use 'testng` or something?

Reply:
Agree that ```java``` may not be the most appropriate profile. Will change it to ```testng```.

PS- This maven profile will be wrapped in a script similar to ```build_framework```.

Reply:
sounds good. I think you mean `run_tests`

Reply:
I was referring to abstracting the actual ```maven``` command by exposing a ```script``` (i.e., wrapping the maven command within the script) to run tests.